pkg/metrics/pool.go

```diff
@@ -0,0 +1,60 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	poolGetCounterMetricsName = "pool_get_total"
+	poolPutCounterMetricsName = "pool_put_total"
+	poolSizeMetricsName       = "pool_size"
```

Comment:
I don't understand the function of this metric. I think we can judge whether a pool is sufficient based on the first two indicators,

Reply:
Um, it's redundant. I will remove it.